sw/device/silicon_creator/mask_rom/BUILD

```diff
@@ -274,3 +279,23 @@ opentitan_functest(
         ],
     ),
 )
+
+# TODO: Add verilator test
+opentitan_functest(
+    name = "e2e_bootup_no_rom_ext_signature",
+    srcs = [
+        "mask_rom_test.c",
+    ],
+    cw310 = cw310_params(
+        args = [
+            "--exec=\"console -q -t0\"",
```

Comment:
As we discussed / came to the conclusion offline, if we remove this line, then we will not need the "Bootstrap: DONE!" portion of the regex above (on line 266), since the FGPA will be held in reset as the test begins, and will not start executing SW until after the test has been bootstrapped into the chip.

Reply:
This command has the side effect of initializing the uart, which will make the host system capture and keep any characters emitted by the DUT between the end of the bootstrap command finishing and the next console command starting.

Either way, I don't think we need "Bootstrap: DONE!" as part of the string.